docs/source/development-guide/schema-deployment/deploy-test-process.rst

```diff
@@ -0,0 +1,15 @@
+Deployment Test Process
+============
+
+Since Augur is a data centric system, software testing is not sufficient for ensuring the integrity of current and future implementations. We also test data collection against real repository lists for each release. 
+
+1. New developers will work on forks in their own repositories, generally. For for active contributors, we may create branches for everyone. 
```

Comment:
Augur should adopt a feature-branch workflow for active contributors that are pushing the main repository instead of contributor-specific branches.